app/src/main/java/org/fossasia/openevent/app/common/data/repository/EventRepository.java

```diff
@@ -94,4 +99,19 @@ public EventRepository(IUtilModel utilModel, IDatabaseRepository databaseReposit
             .build();
     }
 
+    @NonNull
+    @Override
+    public Observable<Event> updateEvent(Event event) {
+        if (!utilModel.isConnected()) {
+            return Observable.error(new Throwable(Constants.NO_NETWORK));
+        }
+
+        event.setTickets(null);
```

Comment:
Actually draft can't be in any of the other category as it is not published yet. I think this would be more convenient to the user than having drafts in upcoming category. What do you say? @iamareebjamal